ci/win32/create_venv.cmd

```diff
@@ -3,6 +3,7 @@ rem @echo off
 
 pushd .
 cd /D "%~dp0"
+py -3.11 -m pip install virtualenv
```

Comment:
Why add this, this doe snot seem to have anything to do with the PR and is it required?